unfurl/server.py

```diff
@@ -0,0 +1,100 @@
+import click
+from flask import Flask, request, jsonify
+from flask_caching import Cache
+import uvicorn
+
+from unfurl.localenv import LocalEnv
+
+
+flaskConfig = {
+    # Use in-memory caching, see https://flask-caching.readthedocs.io/en/latest/#built-in-cache-backends for more options
+    "CACHE_TYPE": "simple", 
+}
+unfurlConfig = {}
```

Comment:
saving global state at the module level is ugly/error prone -- what if we set this on the `app` object? I assume there's some way for the hooks and handlers to get the `app` object? I guess it might be that different but it would look cleaner to me...